common-odm.mk

```diff
@@ -113,6 +113,17 @@ PRODUCT_PACKAGES += \
     libqsocket \
     qmuxd
 
+ifneq ($(TARGET_LEGACY_KEYMASTER),true)
+ifneq ($(TARGET_KEYMASTER_V4),true)
+# Keymaster
+PRODUCT_PACKAGES += \
+    android.hardware.keymaster@3.0-impl-qti
+endif
```

Comment:
please move all keymaster qti blobs !

Reply:
The `-service-qti` "products" aren't actually blobs, they're "virtual" packages as defined in [`rootdir/Android.mk`](https://github.com/sonyxperiadev/device-sony-common/blob/fd3e6c8c993d3aa7ef7ae9856d37dc09d4bbcf3f/rootdir/Android.mk#L27), so they do not belong in the odm image creation file.

We need those "virtual" packages in _every_ build since they supply the `-service-qti.rc` files in `/vendor/etc/init/`.